etcd/cluster-etcd-operator.md

```diff
@@ -0,0 +1,331 @@
+```
+apiVersion: v1
+items:
+- apiVersion: operator.openshift.io/v1
+  kind: Etcd
+  metadata:
+    annotations:
+      release.openshift.io/create-only: "true"
+    creationTimestamp: "2019-10-02T12:38:41Z"
+    generation: 2
+    name: cluster
+    resourceVersion: "108019"
+    selfLink: /apis/operator.openshift.io/v1/etcds/cluster
+    uid: 905931e7-e511-11e9-8e06-020bc572eaf2
+  spec:
+    forceRedeploymentReason: ""
+    logLevel: ""
+    managementState: Managed
+    observedConfig:
+      cluster:
+        members:
+        - name: etcd-member-ip-10-0-154-109.us-east-2.compute.internal
+          peerURLs: https://etcd-0.sbatsche.devcluster.openshift.com:2380
+          status: Ready
+        - name: etcd-member-ip-10-0-164-65.us-east-2.compute.internal
+          peerURLs: https://etcd-2.sbatsche.devcluster.openshift.com:2380
+          status: Ready
+        pending:
+        - name: etcd-member-ip-10-0-135-168.us-east-2.compute.internal
+          peerURLs: https://etcd-1.sbatsche.devcluster.openshift.com:2380
+          status: Remove
+```
+MemberCondition Matrix:
+
+| Condition | Bucket | Status | Result |
+| :---         |     :---:      |     :---:      | :---          |
+| etcd-member container is in CrashLoopBackoff | pending | Remove | Scale down member, stop Pod, remove data-dir, scale up member, and start Pod |
+| etcd-metric container is in CrashLoopBackoff | member | Degraded | Restart Pod |
+| Node reboots | pending | Unknown | No scaling action takes place |
+| Node in network partition | pending | Unknown | No scaling action takes place |
+| New static Pod starts for first time | pending | Add | Scale member up |
+| Previously CrashLoop Pod is started | pending | Unknown | After the Pod is observed as not actively in CrashLoopBackoff status changes to Add |
+| Node removed | pending | Remove | Scale member down |
+| Node added | pending | Add | Scale member up |
+| All containers Ready | member | Ready | No scaling action takes place |
+
+### User Stories
+
+#### Story 1
```

Comment:
I think we're missing a story around a more responsive install with less snowflakes